packages/relay/src/lib/services/hapiService/hapiService.ts

```diff
@@ -250,6 +246,19 @@ export default class HAPIService {
     return client;
   }
 
+  private initPrivateKey(operatorMainKey: string): PrivateKey {
```

Comment:
I would change the name of this method to like `resolvePrivateKey` or something like that it's just like it's not initiating a private key but instead it kinda resolves the string key to `PrivateKey` type? Hmm I'm not entirely sure about this tbh 😓

Reply:
giving good names to variables and methods is one of the hardest things in development. I would suggest:
`resolvePrivateKeyBasedOnFormat`, maybe is long, but describes what the method is doing 😅

Reply:
Done.